Description: Remove unnecessary use of the TemplateHaskell extension
 The patch is not submitted upstream, as the change is very
 simple. The bug has been reported upstream, though.
Bug: https://github.com/vasylp/hgettext/issues/9
Origin: commit:586e11c10910538ae754e25e3f426f82fd0770b9
Forwarded: no
Author: Joachim Breitner <nomeata@debian.org>
Reviewed-by: Sven Bartscher <sven.bartscher@weltraumschlangen.de>
Last-Update: 2015-12-27

Index: hgettext-0.1.30/hgettext.cabal
===================================================================
--- hgettext-0.1.30.orig/hgettext.cabal	2015-06-15 22:26:25.801060283 +0200
+++ hgettext-0.1.30/hgettext.cabal	2015-06-15 22:27:57.632493244 +0200
@@ -24,7 +24,6 @@
 
 Executable hgettext
         Main-Is:                hgettext.hs
-        Extensions:             TemplateHaskell
         Hs-Source-Dirs:         src        
         Build-Depends:          base>=3.0.3.0 && <5, uniplate, haskell-src-exts
         Other-Modules:          Paths_hgettext
